src/client/dfuse/ops/fgetattr.c

```diff
@@ -27,9 +30,17 @@ dfuse_cb_getattr(fuse_req_t req, struct dfuse_inode_entry *ie)
 
 	ie->ie_stat = attr;
 
+	if (save)
+		dh_hash_try_decrefx(fs_handle, save);
+
```

Comment:
is this on purpose, firstly try_decrefx before reply?

Reply:
Yes.  If it can happen first then it can be lockless, if it has to happen after then it needs a write lock.  This is the same as replying to lookup.  This is covered by the large comment in dfuse_fuseops.c which is key to how this PR is supposed to improve performance but that comment is probably in the wrong place.